interfaces-main/src/Employee.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+
+public class Employee {
+   
+    String name;
+    int age;
+    int salary;
+    ArrayList<Employee> bossof;
+
+    Employee(String name, int age, double salary, ArrayList<Employee> bossof){
+        this.name = name;
+        this.age = age;
+        this.salary = (int) salary;
+        this.bossof = bossof;
+    }
+
+
+    public Integer EldersRecursive(ArrayList<Employee> bossofList) {
```

Comment:
Java style code guide uses camel case lower for function names https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names